deployments/automated-previews.mdx

```diff
@@ -207,3 +207,51 @@ destroy_preview:
     - if: $CI_MERGE_REQUEST_ID
       when: manual
 ```
+
+## BitBucket Pipelines
+
+### Creating and cleaning up preview environments
+
+This job can be pasted into your `bitbucket-pipelines.yml` at the root of your repository. Be sure to add repository variables for the ones mentioned in the warning below.
+
+On pull request creation, an Architect environment will be created. The component specified in the repository will be registered with the Architect Cloud and
+deployed to the environment. The pipeline provides the option to remove the deployed component and its environment from the Architect Cloud manually,
+which should be done once the pull request is merged or closed.
+
+<Warning>
+  This example assumes that the repo has `ARCHITECT_EMAIL`, `ARCHITECT_PASSWORD`, `ARCHITECT_ACCOUNT`, and `ARCHITECT_CLUSTER` set as CI/CD variables.
+</Warning>
+
+<Warning>
+  `ARCHITECT_PASSWORD` must be a [personal access
+  token](/deployments/personal-access-tokens).
+</Warning>
```

Comment:
1. This should use `<Note>` instead of warning since its not actually a warning.
2. We should remove the second one and note that fact somewhere else in the doc since that's a relevant statement to all the pipelines. Also, do we _require_ it be a PAT, or is that just the only answer when you have 2FA enabled or are using google/github auth? Might not be relevant to this doc.

Reply:
> We should remove the second one and note that fact somewhere else in the doc since that's a relevant statement to all the pipelines.

I don't agree with moving that block to somewhere else in the doc, because people will likely jump right to their CI/CD provider's section and miss wherever this would move to, then end up confused when the example doesn't work

> Also, do we require it be a PAT, or is that just the only answer when you have 2FA enabled or are using google/github auth?

A user can't log in using their password on the command line, and I wouldn't suggest that they do otherwise in a CI/CD job

Reply:
> I don't agree with moving that block to somewhere else in the doc, because people will likely jump right to their CI/CD provider's section and miss wherever this would move to, then end up confused when the example doesn't work.

I think if it can NEVER be a password then we should consider using a different env variable name. Seems very misleading.

Reply:
I definitely agree and that's felt wrong for a while. I'll add a ticket to allow something more accurate

Reply:
I confirmed that it only takes tokens, and not passwords; I'll update the wording here; There's also an update in the CLI as well since the `--help` command on `architect login` is also misleading